packages/terra-date-time-picker/src/DateTimePicker.jsx

```diff
@@ -144,17 +150,19 @@ const defaultProps = {
   timeInputAttributes: undefined,
   value: undefined,
   timeVariant: DateTimeUtils.FORMAT_24_HOUR,
+  initialTimeZone: DateTimeUtils.getLocalTimeZone(),
```

Comment:
Do we still want to default this to the local timezone or leave it undefined?

Reply:
We could get rid of it and adjust the test examples. Setting it to `undefined` will still set it to local timezone [here](https://github.com/cerner/terra-framework/pull/1044/files#diff-0a8a568512ba2dc166f3b71e47749752R260)